modules/.submodules.json

```diff
@@ -16,7 +16,11 @@
     "zeotapIdPlusIdSystem",
     "haloIdSystem",
     "quantcastIdSystem",
+<<<<<<< HEAD
+    "deepintentDpesIdSystem",
+=======
     "nextrollIdSystem",
+>>>>>>> 81fc1ce9b8471cadb33ebc10a270e8c6d001ea98
```

Comment:
Can you please explain the concept here. Is it constructing a bit shifting hash?

Reply:
Hey @ChrisHuie  are you referring to what Deepintent module does ?

Reply:
I see @ChrisHuie I think i forgot one conflict resolution commit. just pushed it should be good now

Reply:
thanks :)